### PR TITLE
change federation fields/types

### LINKED ISSUE #267: Remove verification_authority_name field
Remove from
model/exposure.go
remove from federation in/out

### LINKED ISSUE #270: federation clean up
- Move transmission risk to an integer
- remove verification authority

### PR DESCRIPTION
**Pull request by mikehelmick**
Remove verification authority name from DB and federation
Change transmission risk to int instead of enum in federation

fixes #270 
fixes #267

### CODE REVIEW

**Comment by mikehelmick** [comment]
/hold

for @squee1945 to review

File: internal/pb/federation.proto [hunk 1]
```diff
@@ -39,16 +39,8 @@ message ContactTracingResponse {
 }
 
 message ContactTracingInfo {
-	// TODO(squee1945 | mikehelmick): this should be an int now.
-	TransmissionRisk transmissionRisk = 1;  // required
+	int32 transmissionRisk = 1;  // required
```
Comment (squee1945) [inline internal/pb/federation.proto:42]: Why did you remove the enum? Having it as an enum makes the values self-describing.

Comment (squee1945) [reply internal/pb/federation.proto:42]: [starting review]

Comment (mikehelmick) [reply internal/pb/federation.proto:42]: because transmission risk has moved to an integer, between 1-8 where the semantic meaning is not handled by the server

**Comment by mikehelmick** [comment]
> Can you add a comment in the proto indicating acceptable values? Can we add validation somewhere?

done.
